charts/zitadel/templates/pdb.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.pdb.enabled }}
+{{- if .Values.pdb.apiVersion }}
+apiVersion: {{ .Values.pdb.apiVersion }}
+{{- else }}
+apiVersion: policy/v1beta1
+{{- end }}
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "zitadel.fullname" . }}
+  labels:
+    {{- include "zitadel.labels" . | nindent 4 }}
+spec:
+  {{- if .Values.pdb.minAvailable }}
+  minAvailable: {{ .Values.pdb.minAvailable }}
+  {{- end }}
+  {{- if .Values.pdb.maxUnavailable }}
+  maxUnavailable: {{ .Values.pdb.maxUnavailable }}
+  {{- end }}
+  selector:
```

Comment:
Is it possible to configure 0 for minAvailable? Because 0 is the go zero value, right?